src/main/kotlin/cacophony/controlflow/X64Register.kt

```diff
@@ -0,0 +1,30 @@
+package cacophony.controlflow
+
+enum class X64Register(val isCallPreserved: Boolean) {
```

Comment:
maybe `Register.Fixed` could be parametrized by such enum, in #124 I added `id: String` but it is a temporary solution

Reply:
I think there is no need for that, as we are most probably going to stick to x86_64 architecture.

Reply:
I mean, we can put this enum in `Register.Fixed` instead of `String`

Reply:
Done